Source/Core/Core/IOS/Network/KD/NWC24DL.cpp

```diff
@@ -125,6 +125,68 @@ bool NWC24Dl::IsRSASigned(u16 entry_index) const
   return !Common::ExtractBit(Common::swap32(m_data.entries[entry_index].flags), 2);
 }
 
+bool NWC24Dl::SkipSchedulerDownload(u16 entry_index) const
+{
+  // Some entries can be set to not be downloaded by the scheduler.
+  return !!Common::ExtractBit(Common::swap32(m_data.entries[entry_index].flags), 5);
+}
+
+bool NWC24Dl::HasSubtask(u16 entry_index) const
+{
+  switch (m_data.entries[entry_index].subtask_type)
+  {
+  case 1:
+  case 2:
+  case 3:
+  case 4:
+    return true;
+  default:
+    return false;
+  }
+}
+
+bool NWC24Dl::IsSubtaskDownloadDisabled(u16 entry_index) const
+{
+  return !!Common::ExtractBit(Common::swap16(m_data.entries[entry_index].subtask_flags), 9);
+}
+
+bool NWC24Dl::IsValidSubtask(u16 entry_index, u8 subtask_id) const
+{
+  return !!Common::ExtractBit(m_data.entries[entry_index].subtask_bitmask, subtask_id);
+}
+
+u32 NWC24Dl::GetNextDownloadTime(u16 record_index) const
+{
+  // Timestamps are stored as a UNIX timestamp but in minutes. We want seconds.
+  return Common::swap32(m_data.records[record_index].next_dl_timestamp) * 60;
```

Comment:
Dumb but can you move the 60 to a file local static constexpr called `SECONDS_IN_MINUTE`, feels like it'd be a bit more telling than having all these 60 numbers around.